### PR TITLE
Minor changes...

### PR DESCRIPTION
**Pull request by mwmentor**
Put in title and words for image...

### CODE REVIEW

File: vuepress/docs/refarch/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# Title2
-
+# DDD-Outline
+Here is a general outline of the BCs in use for Mojaloop 2.0
```
Comment (mdebarros) [inline vuepress/docs/refarch/README.md:2]:
```suggestion
Here is a general outline of the BCs in use for Mojaloop 2.0,
```

**Review by mdebarros — CHANGES_REQUESTED** [review]
Looks good, but minor see my suggested grammar fix.

**Review by mdebarros — APPROVED** [review]
+1

**Comment by mwmentor** [comment]
Merged new changes, and corrections...

**Pull request merged**